packages/kotti-ui/source/kotti-table/components/TableBodyLoadingRow.ts

```diff
@@ -20,9 +20,14 @@ export const TableBodyLoadingRow: any = {
 		const { colSpan, render } = this
 
 		return h('tr', {}, [
-			h('td', { domProps: { colSpan }, class: 'kt-table__loader' }, [
-				render(h),
-			]),
+			h(
+				'td',
+				{
+					domProps: { colSpan },
```

Comment:
https://github.com/vuejs/vue/blob/13f4e7dc03e2caed900ac70ff8b8fe58dda45663/packages/server-renderer/src/util.ts#L6

Reply:
fixed and verified in a local build